core-bundle/src/Resources/contao/dca/tl_content.php

```diff
@@ -1074,6 +1080,87 @@ public function preserveReferenced()
 		}
 	}
 
+	/**
+	 * Filter the content elements
+	 */
+	public function filterContentElements()
+	{
+		if ($this->User->isAdmin)
+		{
+			return;
+		}
+
+		if (empty($this->User->elements))
+		{
+			$GLOBALS['TL_DCA']['tl_content']['config']['closed'] = true;
```

Comment:
should we also set `notEditable` etc. here?

Reply:
Yes. Added in 607c0a219894d089f77a0f59088884ff501ccb1c.